util_cuda.h

```diff
@@ -53,3 +54,9 @@ static void start_gpu_prof() {
 static void stop_gpu_prof() {
     cudaProfilerStop();
 }
+
+static size_t num_blocks(size_t n, size_t threads) {
+    return (n + threads - 1)/threads;
+}
+
+#define cuda_api(f, ...) do { check_status(f(__VA_ARGS__)); } while (0)
```

Comment:
Can we wrap each individual function instead? It's clearer and fits in better with the rest of this file.

Reply:
We can, it's a left-over from a previous iteration where errors included the function name.

Reply:
I moved that macro over to graphs.hcu. There a lot of different CUDA API functions are used once each.